_build/templates/default/sass/_navbar.scss

```diff
@@ -128,7 +128,7 @@
     position: absolute;
     content: ' ';
     right: 12px;
-    top: 24px;
+    top: 40%;
```

Comment:
Maybe you could use `top:50%; transform:translateY(-50%)` to make it vertical centered.